allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/ScenarioUtils.java

```diff
@@ -0,0 +1,24 @@
+package io.qameta.allure.cucumberjvm;
+
+import gherkin.formatter.model.Scenario;
+
+/**
+ * Scenario Utils.
+ */
+@SuppressWarnings("PMD.DefaultPackage")
+final class ScenarioUtils {
+
+    private static final String UNSAFE_CHARACTERS_PATTERN = "[^0-9a-zA-Zа-яА-Я_\\-.]";
+
+    private ScenarioUtils() { }
+
+    /**
+     * Generate safe UUID from Cucumber Scenario ID.
+     * @param scenario Cucumber Scenario instance
+     * @return generated UUID
+     */
+    /* default */ static String getScenarioUuid(final Scenario scenario) {
```

Comment:
Just a note - UUID is an entity with a certain [format](https://en.wikipedia.org/wiki/Universally_unique_identifier), It seems to me that the result of this function is not formatted in a UUID way - which is fine, but would be more readable if the utility would be named otherwise. Personally I make some assumptions seeing the term UUID.

Reply:
I picked a name to match it in line with already existing StepUtils.getStepUuid(?) method. If would need to change it, I will - that's not a problem :)